.ci/run.sh

```diff
@@ -5,7 +5,7 @@ env
 
 set -ex
 
-jruby -rbundler/setup -S rspec -fd
+bundle exec rspec --format=documentation
```

Comment:
Oops! I also added this with my ongoing [PR](https://github.com/logstash-plugins/logstash-input-beats/pull/461).

Reply:
ok. I close this one